test/test_torch.py

```diff
@@ -15003,6 +15003,33 @@ def test_cat_big(self, device):
         result = torch.cat(concat_list)
         self.assertEqual(result.size(0), SIZE1 + SIZE2)
 
+    @onlyCPU
+    def test_cat_bad_dtypes(self, device):
```

Comment:
why is this onlyCPU?

Reply:
Oh I was testing it locally w/o GPU -- forgot to change back